### DISCUSSION TITLE
Feature Request: Manage Hostname process to validate DNS configuration and show details of any misconfiguration

### DISCUSSION THREAD

**Discussion by lakesol** [umbraco/Umbraco.Cloud.Issues#803] [Ideas]
### Issue description

We recently had an issue where DNS was causing CAA DNS requests to fail (SERVFAIL) and as a result cloudflare could not issue renewed certificates for some of our hostnames.

The issue was resolved by correcting the DNS server with the client.

The diagnosis of this was complex and involved a considerable amount of time by umbraco support and cloudflare.

As an enhancement, can the manage hostname dashboard be updated to include a DNS config check, this would ensure the DNS is;

- Pointing to a correct IP address - A record, where CNAME support is not supported by the DNS provider
- Pointing to the correct CNAME - A record, where CNAME support is supported by the DNS provider
- DNS CAA status - ensure dig caa <hostmame> returns an appropriate (not fail) result
- Any other checks that can be carried out to preempt hostname/certificate/configuration issues

**Comment by sajumb** [comment]
Thanks for the feature request!
